e2etest/pkg/mac/mac.go

```diff
@@ -105,3 +117,69 @@ func UpdateNodeCache(nodes []corev1.Node, exec executor.Executor) error {
 
 	return nil
 }
+
+// AddrResolutionReq does an ARP/NS request for the given IP.
+func AddrResolutionReq(ip string, exc executor.Executor) error {
+	var cmd string
```

Comment:
Move these below, near where you use them.

Reply:
done